elasticdl/python/tests/k8s_instance_manager_test.py

```diff
@@ -18,7 +18,7 @@
 from unittest.mock import MagicMock, call
 
 from elasticdl.python.common.k8s_client import PodType
-from elasticdl.python.master.k8s_instance_manager import InstanceManager
+from elasticdl.python.master.pod_manager import PodManager
```

Comment:
I will rename k8s_instance_manager_test to pod_manager_test in the next PR.
In the current PR, I keep the original file name to check the difference of the file content.